src/screens/regimen/RegimenScreen/cards/Card.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { Foundation } from '@expo/vector-icons';
+
+import { useThemeContext } from '../../../../theme';
+import Image from '../../../../libs/responsiveImage/index';
+import applyScale from '../../../../utils/applyScale';
+import {
+  CardContainer,
+  Context,
+  Time,
+  ResponsiveImage,
+  Text,
+  Heading,
+  Content
+} from './style';
+
+interface CardProps {
+  time: string;
+  order: number;
+  image: string;
+  content: string;
+  heading: string;
+}
+
+export default function Card(props: CardProps) {
+  const [order, setOrder] = useState<'row' | 'row-reverse'>('row');
+  const { fonts, colors } = useThemeContext();
+
+  useEffect(() => {
+    if (props.order === 1) {
```

Comment:
move this logic of determining position of the image to the parent of this card. Let this card be responsible to only display data not making a decision that causes re-rendering each time its been used.

Reply:
Please check the current one if it fits better